superbench/benchmarks/micro_benchmarks/ib_validation_performance/ib_validation_performance.cc

```diff
@@ -301,7 +303,7 @@ float run_cmd(string cmd_prefix, int port, bool server, string hostname) {
 }
 
 // The ranks in vector of (server, client) run commands parallel
-vector<float> run_cmd_parallel(string cmd_prefix, const vector<std::pair<int, int>> &run_pairs_in_parallel,
+vector<float> run_cmd_parallel(string cmd_prefix, int timeout, const vector<std::pair<int, int>> &run_pairs_in_parallel,
```

Comment:
Why not add the end of this function? What's your thinking here?

Reply:
the first two arguments are parsed from boost program options directly. the last one is generated in other functions